sherpa/csrc/offline-nemo-enc-dec-ctc-model-bpe.cc

```diff
@@ -46,4 +38,17 @@ torch::Tensor OfflineNeMoEncDecCTCModelBPE::GetLogSoftmaxOutLength(
   return {};
 }
 
+void OfflineNeMoEncDecCTCModelBPE::WarmUp(torch::Tensor features,
+                                          torch::Tensor features_length) {
+  // For Citrinet, the subsampling_factor_ is 16.
```

Comment:
For Citrinet the subsampling factor is 8x, for conformer 4x

Reply:
Ah, yes, you are right. 

Thanks.